content/tutorial-paychan.md

```diff
@@ -0,0 +1,489 @@
+                    "settle_delay": 86400
+                }],
+                "status": "success"
+            }
+        }
+
+    The payee should check the following:
+
+    - Confirm the `destination_account` field has the payee's correct address.
+    - Confirm the `settle_delay` field has a settlement delay in seconds that provides enough time for the payee to redeem outstanding claims.
+    - Confirm the fields `cancel_after` (immutable expiration) and `expiration` (mutable expiration), if they are present, are not too soon. The payee should take note of these times so they can be sure to redeem claims before then.
+    - Take note of the `public_key` and `channel_id` fields. These are necessary later to verify and redeem claims.
+    - _(Optional)_ Confirm the `destination_tag` field, if present, has the correct destination tag (if desired).
+
+    Since there can be multiple channels between the same two parties, it is important for the payee to check the qualities of the correct channel. If there is any chance of confusion, the payer should clarify the Channel ID (`channel_id`) of the channel to use.
+
+3. **The payer creates one or more signed _claims_ for the XRP in the channel.** The amounts of these claims depends on the specific goods or services the payer wants to pay for.
+
+    Each claim must be for a cumulative amount. In other words, to purchase two items at 10 XRP each, the first claim should have an amount of 10 XRP and the second claim should have an amount of 20 XRP. The claim can never be more than the total amount of XRP allocated to the channel. (A [PaymentChannelFund][] transaction can increase the total amount of XRP allocated to the channel.)
+
+    You can create claims with the `channel_authorize` API method. The following example authorizes 1 XRP from the channel:
+
+    Request:
+
+        POST http://localhost:5005/
+        Content-Type: application/json
+
+        {
+            "method": "channel_authorize",
+            "params": [{
+                "channel_id": "5DB01B7FFED6B67E6B0414DED11E051D2EE2B7619CE0EAA6286D67A3A4D5BDB3",
+                "secret": "s████████████████████████████",
+                "amount": "1000000"
+            }]
+        }
+
+    Response:
+
+        {
+            "result": {
+                "signature": "304402204EF0AFB78AC23ED1C472E74F4299C0C21F1B21D07EFC0A3838A420F76D783A400220154FB11B6F54320666E4C36CA7F686C16A3A0456800BBC43746F34AF50290064",
+                "status": "success"
+            }
+        }
+
+
+4. **The payer transmits a claim to the payee as payment for goods or services.**
+
+    This communication happens "off-ledger" in any communication system the payer and payee can agree to. It's recommended that the two parties use secure communications for this, but it's not strictly necessary. Only the payer or payee of a channel can redeem claims against that channel.
```

Comment:
I like what you're getting at here, but I'm going to avoid "technically" here because taking security precautions is almost never "technically" necessary. (Just hope you'll get lucky and not pwned!)